app/controllers/api/v4/converts_controller.rb

```diff
@@ -7,6 +7,13 @@ def create
 
     @run = Run.new(run_file: run_file, user: nil)
     @run.parse(fast: ["on", "1"].include?(params[:historic]) ? false : true, convert: true)
+    unless @run.splits.present?
```

Comment:
You can store the result of `@run.parse` and see if it's there after parsing:

```ruby
parse_result = @run.parse(:stuff)
unless parse_result.present?
  # do things
end
```

If that's too verbose you could simplify it to

```ruby
unless @run.parse(:stuff).present?
  # do things
end
```

If you want to be more idiomatic you could even change `Run#parses?` to forward its args on to `Run#parse` so that you could do

```ruby
unless @run.parses?(:stuff)
  # do things
end
```